index.html

```diff
@@ -4,9 +4,234 @@
+						</a>
+					</li>
+				</ul>
+			</nav>
+		</div>
+		<div class="headline">
+			<h3>
+				"The biggest International health care trade show in the east African region"
+			</h3>
+			<h1>
+				ETHIO HEALTH <br>EXHIBITION AND CONGRESS 2023
+			</h1>
+			<p class="headline-paragraph-description">
+				Ethio Health is the largest gathering of healthcare and trade professionals in the country. The 2023 edition of the
+				event is expected to welcome nearly 100 exhibiting companies and 4,000 + attendees.
+			</p>
+			<h2 class="headline-date">
+					2023.02.23(THU) ~ 25(SAT)
+			</h2>
+			<p class="headline-paragraph-location">
+					@ Ethiopian Skylight Hotel
+			</p>
+		</div>
+	</div>
+</header>
+<main>
+	<div class="container">
+		<div class="main-title">
+			<p>Main Program</p>
+		</div>
+		<div class="main-program-container">
+			<div class="main-cards">
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="./img/icon/lecture-icon.png">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Lecture
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Listen to the speakers from various countries about the messeges of sharing and opening
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/exhibit-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Exhibition
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Appreciate various creations applying CC license of artists, organized from Art Center
+							Nabi.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/forum-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Forum
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Have the time to share your thoughts and opinions with experts for each topic.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/workshop-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Workshop
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Try creating your own work using open source license rather than just watching at it.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/discussion.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Discussion
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Get opportunities to network with CC affiliates around the world, also after the summit.
+						</p>
+					</div>
+				</div>
+			</div>
+		</div>
+		<button type="button" class="main-btn">
+			Join Ethio Health Exibition 2023
+		</button>
```

Comment:
- [ ] Please make sure that the button here looks exactly like the design (and by look exactly I mean that the text should be in one line at each screen size) at each screen size you can add a responsive size for the font size like `vw` or you can make use of the media query. 
![image](https://user-images.githubusercontent.com/74744787/218277207-729a6af5-6e7d-4794-86bb-a91670738fb8.png)